tools/opt/opt.cpp

```diff
@@ -239,6 +239,10 @@ Options (in lexicographical order):
   --print-all
                Print SPIR-V assembly to standard error output before each pass
                and after the last pass.
+  -ftime-report
+               Print the resource utilization of each pass (e.g., CPU time,
+               RSS). Currently it supports only Unix systems. This option is
+               same with -ftime-report in GCC.
```

Comment:
is the same as -ftime-report in GCC.

Reply:
Should we mention that it will be output to stderr and not stdout?

Reply:
Yes, that's right. I updated it.

Reply:
Done